website/frontend/src/types/common.ts

```diff
@@ -5,14 +5,14 @@ export type PaginationModel = {
   limit?: number;
   offset?: number;
   type?: string;
-  term?: string;
+  query?: string;
 };
 
 export type UserPrograms = {
   owner: string | null;
   limit?: number;
   offset?: number;
-  term?: string;
+  query?: string;
```

Comment:
Are you sure that it's optional?

Reply:
YesThis param is still **term** on the server side

Reply:
Then here it should be `term` too